.github/workflows/docker-test.yaml

```diff
@@ -16,6 +16,7 @@ on:
       - master
       - production
     paths:
+      - '**/**'
```

Comment:
@vpetersson, it seems like the execution of unit tests are skipped, even though changes were made on Python files, so I've introduced this change.